audbackend/core/filesystem.py

```diff
@@ -75,32 +89,46 @@ def _get_file(
 
     def _ls(
             self,
-            folder: str,
+            path: str,
     ):
-        r"""List all files under folder."""
+        r"""List all files under (sub-)path."""
+
+        if path.endswith('/'):  # find files under sub-path
+
+            path = self._expand(path)
+            if not os.path.exists(path):
+                utils.raise_file_not_found_error(path)
+            paths = audeer.list_file_names(path, recursive=True)
 
-        folder = self._folder(folder)
+        else:  # find versions of path
 
-        if not os.path.exists(folder):
-            utils.raise_file_not_found_error(folder)
+            root, _ = self.split(path)
+            root = self._expand(root)
+            vs = audeer.list_dir_names(
+                root,
+                basenames=True,
```

Comment:
And here the same

```suggestion
                basenames=True,
                hidden=True,
```

Reply:
Here `True` is the default value, but it doesn't hurt to be verbose of course. Maybe we'll change it someday in `audeer`.